src/main/java/com/transportcompany/transportcompany/models/entities/Vehicle.java

```diff
@@ -0,0 +1,41 @@
+package com.transportcompany.transportcompany.models.entities;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Table;
+
+@Entity
+@Table (name = "vehicle")
+
```

Comment:
We should not have an empty row between the class declaration and the class annotations.